README.md

```diff
@@ -170,6 +170,70 @@ the recommended order of reading:
+* INSTALL-UBUNTU - Only for Ubuntu (Please read INSTALL for general installation information)
+* Kerberos-Auditing-HOWTO - how to audit Kerberos hashes
+* MARKOV - basic information/usage for the Markov mode
+* MASK - Information on mask mode and examples
+* NETNTLM_README - LM/NTLM Challenge / Response Authentication
+* OFFICE - JtR on Office 2003 / 2007 / 2010 / 2013 files
+* PRINCE - JtR prince mode crash course
+* README-CUDA - JtR CUDA updates/status
+* README-DISTROS - building a CPU-fallback chain (with OpenMP fallback too) for distros
+* README-MIC - how to build JtR for MIC
+* README-OPENCL - how to use opencl in JtR
+* README-PDF - PDF cracking in JtR
+* README-PST - PST cracking in JtR
+* README-ZIP - ZIP cracking in JtR
+* README-ZTEX - using ZTEX with JtR
+* README-krb5-18-23 - kdb5_util in JtR
+* README.7z2john.md - 7z2 credit in JtR
+* README.Apple_DMG - cracking DMG in JtR
+* README.BitLocker - cracking bitlocker in JtR
+* README.Ethereum - cracking etherum wallet in JtR
+* README.FileVault2 - cracking password protected FileVault 2 encrypted volumes in JtR
+* README.FreeBSD(*) - building JtR-jumbo on FreeBSD
+* README.IBM_AS400 - How to get & crack AS/400 hashes
+* README.LUKS - Cracking LUKS passphrases
+* README.LotusNotes - IBM Lotus Domino hash extractor
+* README.MinGW - Fedora >= 22 cross-compiling instructions
+* README.apex - dumping Oracle APEX...
+* README.bash-completion - how to enable bash completion for JtR
+* README.bitcoin - cracking bitcoin wallet files with JtR
+* README.coding-style(*) - accepted coding style for contributors
+* README.cprepair - reading broken files
+* README.format-epi - how to dump EPiServer password hashes
+* README.gpg - PGP Zip / OpenPGP / GnuPG private cracking in JtR
+* README.IOS 7 - cracking IOS 7 restrictions PIN code
+* README.keychain - Cracking Apple's Mac OS Keychain files
+* README.keyring - cracking GNOME Keyring files
+* README.keystore - cracking KeyStore files
+* README.kwallet - cracking KWallet files
+* README.librexgen - howto perform regex expression work within JtR
+* README.mozilla - cracking Mozilla Firefox, Thunderbird and SeaMonkey master passwords
+* README.mpi - using MPI in JtR
+* README.pwsafe - cracking Password Safe 3.x and Password Gorilla databases with john
+* README.ssh - Cracking password protected ssh private keys
+* RULES-hashcat - wordlist rules with hashcat extension
+* Regen-Lost-Salts.txt - regen-lost-salt in JtR
+* SIPcrack-LICENSE - the SIPcrack license
+* SecureMode-tutorial.md - using JtR's SecureMode feature
+* dynamic_history.txt - upto date history on dynamic_fmt.c file
+* john-1.7.9-jumbo-7-licensing-stats.txt - license status for john-1.7.9-jumbo-7
+* libFuzzer-HOWTO.txt - how to build libfuzzer
+* pass_gen.Manifest - pass_gen.pl version history
+* pcap2john.readme - all the prior copyright headers from the independent XXX2john.py PCAP conversion utilities
+* AxCrypt-Auditing-HOWTO - auditing AxCrypt secrets
```

Comment:
This document says "recommended order of reading" at the top. Then this commit starts with "AddressSanitizer-HOWTO.txt" which is really a super-niched thing for developers only.

There are so many documents we might need to drop the "recommended order" clause, or break it with a "rest of documents in alphabetical (or whatever) order"

Reply:
@magnumripper hahahaha. Right. I will rearrange it. But i think with the readme cracking stuff, I can create another little for that and group them in alphabetical order.